lib/tasks_private/instance_types.rake

```diff
@@ -52,58 +49,55 @@ task :instance_types => [:environment] do
-    %i(family instance_store_size instance_store_volumes vpc_only enhanced_networking).each do |key|
-      o[key] = i[key]
+  def format_instance_miq(i)
+    {}.tap do |o|
+      # x1.16large manually fixed to x1.16xlarge
+      # i3.16large manually fixed to i3.16xlarge
+      %i(family instance_store_size instance_store_volumes vpc_only enhanced_networking).each do |key|
+        o[key] = i[key]
+      end
+      o[:memory] = i[:memory].to_f.gigabyte
+      o[:name] = i[:instance_type]
+      o[:description] = i[:pretty_name]
+      o[:vcpu] = i[:vCPU]
+      o[:ebs_optimized_available] = i[:ebs_optimized]
+      o[:instance_store_volumes] = i.fetch_path(:storage, :devices).to_i
+      o[:instance_store_size] = (o[:instance_store_volumes] * i.fetch_path(:storage, :size).to_f).gigabyte
+      o[:instance_store_ssd] = i.fetch_path(:storage, :ssd)
+      o[:architecture] = i[:arch].map(&:to_sym).sort
+      o[:virtualization_type] = i[:linux_virtualization_types].map(&:downcase).map(&:to_sym).sort.map { |v| v == :pv ? :paravirtual : v }
+      o[:network_performance] =
+        case i[:network_performance]
+        when '10 Gigabit', '20 Gigabit', '25 Gigabit'
+          :very_high
+        when ''
+          :unknown
+        else
+          i[:network_performance].downcase.tr(' ', '_').to_sym
+        end
+      o[:ebs_only] = i[:storage].nil?
+      o[:physical_processor] = i[:physical_processor].nil? ? '' : i[:physical_processor].chomp('*')
+      o[:processor_clock_speed] = i[:clock_speed_ghz]
+      o[:intel_aes_ni] = true # looks like only deprecated types dont support it, so we assume all new types support it
+      #  https://aws.amazon.com/ec2/instance-types/#intel
+      o[:intel_avx] = i[:intel_avx]
+      o[:intel_avx2] = i[:intel_avx2]
+      o[:intel_turbo] = i[:intel_turbo]
+      o[:cluster_networking] = cluster_networking?(i[:instance_type])
     end
-    o[:memory] = i[:memory].to_f.gigabyte
-    o[:name] = i[:instance_type]
-    o[:description] = i[:pretty_name]
-    o[:vcpu] = i[:vCPU]
-    o[:ebs_optimized_available] = i[:ebs_optimized]
-    o[:instance_store_volumes] = i.fetch_path(:storage, :devices).to_i
-    o[:instance_store_size] = (o[:instance_store_volumes] * i.fetch_path(:storage, :size).to_f).gigabyte
-    o[:instance_store_ssd] = i.fetch_path(:storage, :ssd)
-    o[:architecture] = i[:arch].map(&:to_sym).sort
-    o[:virtualization_type] = i[:linux_virtualization_types].map(&:downcase).map(&:to_sym).sort.map { |v| v == :pv ? :paravirtual : v }
-    o[:network_performance] = case i[:network_performance]
-                              when '10 Gigabit', '20 Gigabit'
-                                :very_high
-                              else
-                                i[:network_performance].downcase.tr(' ', '_').to_sym
-                              end
-    # see https://github.com/ManageIQ/manageiq/issues/741#issuecomment-57353290
```

Comment:
I'd leave this comment in the code as a pointer to what 'ebs_only' means

Reply:
sure, done!